server/shared/database/seed/index.js

```diff
@@ -0,0 +1,115 @@
+'use strict'
+
+const Promise = require('bluebird')
+const pick = require('lodash.pick')
+
+const userData = require('./users.json').data
+const dbConnectionData = require('./dbConnections.json').data
+const examData = require('./exams.json').data
+const questionData = require('./questions.json').data
+
+const DIFFICULTY = ['EASY', 'MEDIUM', 'HARD']
+
+function insertAnswers(Answer, examsTaken) {
+  return Promise.map(examsTaken, examTaken => {
+    return examTaken.getStudent().then(student => {
+      return examTaken.getQuestions().then(questions => {
+        return Promise.map(questions, question => {
+          return Answer.create({
+            givenQuery: question.correctQuery, correct: true
+          })
+            .then(answer => answer.setStudent(student))
+            .then(answer => answer.setExam_taken(examTaken))
+            .then(answer => answer.setQuestion(question))
+            .then(() => examTaken.update({ score: 100 }))
+        })
+      })
+    })
+  })
+}
+
+function insertTakenExams(ExamTaken, exams, students) {
+  const result = []
+
+  exams.forEach(exam => {
+    students.forEach(student => {
+      result.push(
+        ExamTaken.create()
+          .then(examTaken => examTaken.setExam(exam))
+          .then(examTaken => examTaken.setStudent(student))
+          .then(examTaken => {
+            return exam.getQuestions()
+              .then(questions => examTaken.setQuestions(questions))
+              .then(() => examTaken)
+          })
+      )
+    })
+  })
+
+  return Promise.all(result)
+}
+
+function insertQuestions(Question, exams, dbConnections) {
+  const result = []
+  let examsLen = exams.length
+
+  questionData.forEach((record, index) => {
```

Comment:
Can use `Promise.map()` here same as `seed/index.js`. No need for `result` mediator.